### PR TITLE
AUTH: Updating author list

### PR DESCRIPTION
**Pull request by shamisp**
This is a port of the list from 1.8.0 plus recent contributors from the
master.

Signed-off-by: Pavel Shamis (Pasha) <pasharesearch@gmail.com>

### CODE REVIEW

**Comment by shamisp** [comment]
@yosefe I hope I didn't miss anybody. Please take a look.

**Comment by shamisp** [comment]
ok to test

File: AUTHORS [hunk 1]
```diff
@@ -1,46 +1,75 @@
...
+Peter Andreas Entschev <peter@entschev.com>
+Peter Rudenko <peterr@mellanox.com>
+Qiang Yu <Qiang.Yu@amd.com>
+Rohit Zambre <rzambre@uci.edu>
 Sasha Kotchubievsky <sashakot@mellanox.com>
-Sergey Oblomov
-Sergey Shalnov
-Serguei Sagalovitch
+Scott Saulters <ssaulters@nvidia.com>
+Sergey Lebedev <sergey.a.lebedev@gmail.com>
```
Comment (yosefe) [inline AUTHORS:52]: `sergeyle@mellanox.com`

File: AUTHORS [hunk 2]
```diff
@@ -1,46 +1,75 @@
...
+Sourav Chakraborty <souchakr@amd.com>
+Srinivasan Subramanian <srinivasan.subramanian@amd.com>
 Stephen Richmond <srichmond@dancer.icl.utk.edu>
 Swen Boehm <boehms@ornl.gov>
 Tony Curtis <anthony.curtis@stonybrook.edu>
+Valentin Petrov <valentinp@mellanox.com>
 Wenbin Lu <wenbin.lu@stonybrook.edu>
 Xin Zhao <xinz@mellanox.com>
 Yossi Itigin <yosefe@mellanox.com>
+Yuriy Shestakov <yshestakov@gmail.com>
```
Comment (yosefe) [inline AUTHORS:67]: `yuriis@mellanox.com`

File: AUTHORS [hunk 3]
```diff
@@ -1,46 +1,75 @@
...
 Guy Shattah <sguy@mellanox.com>
 Hiroyuki Sato <hiroysato@gmail.com>
 Howard Pritchard <howardp@lanl.gov>
-Igor Ivanov <igori@mellanox.com>
-Ilya Nelkenbaum
-Jeff Daily
-John Snyder <jsnyder@cs.duke.edu>
-Khaled Hamidouche
-Manjunath Gorentla Venkata <manjugv@ornl.gov>
+Igor Ivanov <Igor.Ivanov@itseez.com>
```
Comment (yosefe) [inline AUTHORS:24]: `igori@mellanox.com`

**Pull request merged**